### PR TITLE
harness: consume output of balancer when running tests

### LINKED ISSUE #1188: harness: consume output of balancer when running tests
When running harness tests against the balancer, the balancer outputs a ton of logs. It would be nice if the harness could consume these logs and re emit them so that they don't show up in the output unless `--nocapture` is used. Note that `--nocapture` already exists and so implementing that part is not necessary, eg. `cargo test -- --nocapture`

### PR DESCRIPTION
**Pull request by Victor-M-Giraldo**
closes #1188

### CODE REVIEW

**Comment by Victor-M-Giraldo** [comment]
Do you mean something like this? I'm aware it doesn't log it back out to the parent process quite yet.

File: crates/harness/src/test_runner.rs [hunk 1]
```diff
@@ -84,6 +84,10 @@
             .envs(envs)
             .spawn()
             .expect("Failed to start balancer");
+
+        let _stdout = child.stdout.take().expect("Failed to capture stdout.");
+        let _stderror = child.stderr.take().expect("Failed to capture stderror.");
+
```
[the remark this answers is not in the document]
Comment (dyc3) [reply crates/harness/src/test_runner.rs:90]: this is out of scope for this PR

**Comment by dyc3** [comment]
@Victor-M-Giraldo If it prevents stdout and stderr from being logged to the console, yeah. This isn't really enough for me to give meaningful feedback.

**Comment by Victor-M-Giraldo** [comment]
Do you think you might have some time tomorrow anywhere between 9-12 to join a call and help me out with this?

I know the task should be pretty easy, but i'm having some trouble with it.

**Comment by dyc3** [comment]
Yeah I got time at like 10am

**Comment by Victor-M-Giraldo** [comment]
That works. Thank you.

**Review by dyc3 — APPROVED** [review]

**Pull request merged**